config/rukpak/apis/webhooks/resources/deployment.yaml

```diff
@@ -14,6 +17,13 @@ spec:
       labels:
         app: webhooks
     spec:
+      affinity:
+        podAntiAffinity: 
+          requiredDuringSchedulingIgnoredDuringExecution: 
```

Comment:
Could use [preferredDuringSchedulingIgnoredDuringExecution](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity) as well, impacts scheduling strategy.

Reply:
Would obviously not work on SNO clusters.